pkg/webhook/validation/crdvalidation.go

```diff
@@ -0,0 +1,12 @@
+package validation
+
+import "k8s.io/utils/strings/slices"
+
+var (
+	validObjectGroups = []string{"networking.fleet.azure.com", "fleet.azure.com", "multicluster.x-k8s.io"}
+)
+
+// CheckCRDGroup checks to see if the input CRD group is a fleet CRD group.
+func CheckCRDGroup(group string) bool {
```

Comment:
Hi Arvind! I think for this one, as mentioned in the design doc, we might need to use a vague match rather than an exact match.